src/baldr/graphreader.cc

```diff
@@ -1032,17 +1038,5 @@ graph_tile_ptr LimitedGraphReader::GetGraphTile(const GraphId& graphid) {
   return reader_.GetGraphTile(graphid);
 }
 
-const valhalla::IncidentsTile::Metadata&
```

Comment:
moved to `src/thor/triplegbuilder.cc` to have `valhalla::IncidentsTile` fwd in `graphreader.h`